demos/common/python/openvino/model_zoo/model_api/README.md

```diff
@@ -0,0 +1,150 @@
+
+Currently, `OpenvinoAdapter` and `OVMSAdapter` are supported.
+
+#### OpenVINO Adapter
+
+`OpenvinoAdapter` hides the OpenVINO™ toolkit API, which allows Model API wrappers launching with models represented in Intermediate Representation (IR) format.
+It accepts a path to either `xml` model file or `onnx` model file.
+
+#### OpenVINO Model Server Adapter
+
+`OVMSAdapter` hides the OpenVINO Model Server python client API, which allows Model API wrappers launching with models served by OVMS.
+
+Refer to __[`OVMSAdapter`](adapters/ovms_adapter.md)__ to learn about running demos with OVMS.
+
+For OpenVINO Model Server Adapter employment, you need to install the requirements:
+```sh
+pip install <omz_dir>/demos/common/python/requirements_ovms.txt
+```
+
+## Model API Pipelines
+
+Model API Pipelines represent the high-level wrappers upon the input data and accessing model results management.
+It performs the data submission for model inference, verification of inference status, whether the result is ready or not, and results accessing.
+
+The `AsyncPipeline` is available, which handles the asynchronous execution of a single model.
+
+## Ready-to-use Model API solutions
+
+To apply Model API wrappers in custom applications, learn the provided example of common scenario of how to use Python* Model API.
+
+ In the example, the SSD architecture is used to predict bounding boxes on input image `"sample.png"`. The model execution is produced by `OpenvinoAdapter`, therefore we submit the path to the model's `xml` file. The model is loaded on a CPU device inside the adapter.
+
+Once the SSD model wrapper instance is created, we get the predictions by the model in one line: `ssd_model(input_data)` - the wrapper performs the preprocess method, synchronous inference on OpenVINO™ toolkit side and postprocess method.
+
+```python
+import cv2
+from openvino.model_zoo.model_api.models import SSD
+from openvino.model_zoo.model_api.adapters import OpenvinoAdapter, create_core
+
+
+# read input image using opencv
+input_data = cv2.imread("sample.png")
+
+# define the path to mobilenet-ssd model in IR format
+model_path = "public/mobilenet-ssd/FP32/mobilenet-ssd.xml"
+
+# create adapter for OpenVINO™ runtime, pass the model path
+model_adapter = OpenvinoAdapter(create_core(), model_path, device="CPU")
+
+# create model API wrapper for SSD architecture
+ssd_model = SSD(model_adapter, preload=True)
+
+# apply input preprocessing, sync inference, model output postprocessing
+results = ssd_model(input_data)
+
+# visualize results and show it
+image_with_bboxes = draw_detections(input_data, results)
+cv2.imshow('Detection Results', image_with_bboxes)
+```
+
+To study the complex scenarios, refer to [Open Model Zoo Python* demos](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos), where the asynchronous inference is applied.
+
+The list of Open Model Zoo demos with Model API support:
+- BERT Named Entity Recognition Python* Demo
```

Comment:
probably it make sense to provide link on demo readme

Reply:
Added links to the demos